features/embedded-wallets/embedded-wallet.interface.ts

```diff
@@ -0,0 +1,16 @@
+import { Client } from "@xmtp/react-native-sdk";
+import { LocalAccount } from "viem/accounts";
+
+type Account = LocalAccount<any>;
```

Comment:
I feel we use `Account` in so many places. I think we'll need to be more and more specific.